docs/sources/administration/announcement-banner/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+draft: true
+aliases:
+  - ../administration/reports/
+  - ../enterprise/export-pdf/
+  - ../enterprise/reporting/
+  - ../panels/create-reports/
+  - reporting/
+keywords:
+  - grafana
+  - announcement
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Announcement banner
+title: Create and configure announcement banner
+description: Creat a banner to show important updates and information at the top of on every page
+refs:
+  rbac:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/access-control/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/access-control/
+---
+
+# Create and configure announcement banner
+
+Announcement banner allows you to show important updates and information at the top of every page in Grafana. You can use the announcement banner to communicate important information to your users, such as maintenance windows, new features, or other important updates.
+
+## Create or update an announcement banner
+
+Only organization administrators can create announcement banner by default. You can customize who can create announcement banner with [Role-based access control](ref:rbac).
+
+To create or update an announcement banner, follow these steps:
+
+1. Click **Administration > General > Announcement banner** in the side navigation menu.
+
+   The Announcement banner page allows you to view, create and update the settings for a notification banner. Only one banner can be created at a time.
```

Comment:
UI elements and pages in bold https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold.
Consistently use announcement banner instead of notification banner.
```suggestion
   The **Announcement banner** page allows you to view, create, and update the settings for an announcement banner.
   Only one banner can be active at a time.